doc/examples/viz_surfaces.py

```diff
@@ -0,0 +1,99 @@
+"""
+========================================
+Visualize surfaces; load/save, get/set and update vtkPolyData.
+========================================
+
+import usefull functions and dipy utils 
+"""
+from __future__ import division, print_function, absolute_import
+import numpy as np
+
+# and dipy tools
+import dipy.io.vtk as io_vtk
+import dipy.viz.utils as ut_vtk
+from dipy.viz import window
+
+# Conditional import machinery for vtk
+# Allow import, but disable doctests if we don't have vtk
+from dipy.utils.optpkg import optional_package
+vtk, have_vtk, setup_module = optional_package('vtk')
+
+
+"""
+generate a empty vtkPolyData
+"""
```

Comment:
Add an empty line between comments and code. Otherwise the tutorial will not render correctly.